src/app/cart/cart.component.js

```diff
@@ -16,9 +18,10 @@ let componentName = 'cart';
 class CartController{
 
   /* @ngInject */
-  constructor($uibModal, cartService) {
+  constructor($uibModal, cartService, sessionService) {
     this.$uibModal = $uibModal;
     this.cartService = cartService;
+    this.session = sessionService.current;
```

Comment:
It's used in the template current to switch an anchor tag from checkout to sign-in depedning on current session.role.